packages/turndown/src/commonmark-rules.js

```diff
@@ -127,6 +127,20 @@ rules.subscript = {
   }
 }
 
+// Handles foreground color changes as created by the rich text editor.
+// We intentionally don't handle the general style="color: colorhere" case as
+// this may leave unwanted formatting when saving websites as markdown.
+rules.foregroundColor = {
+  filter: function (node, options) {
+    const hexColorRegex = /^\#[a-zA-Z0-9]+$/;
+    return node.nodeName === 'SPAN' && node.getAttribute('joplin-color')?.match(hexColorRegex);
```

Comment:
Perhaps a better option here would be to use a turndown option `perserveTextColors` (similar to what we do with tables).

It may also make sense to put `rules.foregroundColor` somewhere else (`turndown-plugin-gfm`?).

Reply:
Yes I wonder about this too. We can't have it enabled by default as it would apply to email-to-note or clipped web pages, but when switching between the rich text and Markdown I think that would make sense